### PR TITLE
Mailback

### PR DESCRIPTION
**Pull request by Hochart-dev**
add backend for Sending Mail.

### CODE REVIEW

**Review by ThomasSupervil — DISMISSED** [review]
Seems good

**Comment by LoicBrassart** [comment]
Solve conflicts please

**Review by ThomasSupervil — DISMISSED** [review]
resolve conflicts

**Review by ThomasSupervil — DISMISSED** [review]
🥸

File: conf.js [hunk 1]
```diff
@@ -9,4 +10,25 @@ const db = mysql.createPool({
...
+  },
+});
+
+transporter.verify(function (error) {
+  if (error) {
+    console.log(error);
+  } else {
+    console.log('Server is ready to take our messages!');
+  }
+});
```
Comment (LoicBrassart) [inline conf.js]: Useless, remove this

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
A bit of useless code to delete

**Review by LoicBrassart — APPROVED** [review]
SnE!

**Review by ThomasSupervil — APPROVED** [review]
goody

**Pull request merged**